spec/helpers.lua

```diff
@@ -1300,6 +1300,7 @@ end
 -- @return A thread object (from the `llthreads2` Lua package)
 -- @see kill_http_server
 local function http_server(port, opts)
+  print(debug.traceback("[warning] http_server is deprecated, use nginx fixtures.http_mock or helpers.http_mock instead.", 2))
```

Comment:
What is "nginx fixtures.http_mock"?

Reply:
@ADD-SP introduced a way to mock HTTP directly with Nginx (add a mock to the configure fixture). Not sure how to name it.

Reply:
I didn't introduce it, I just enhanced it.

Reply:
Rephrased.